README.md

```diff
@@ -45,152 +21,53 @@ though we include a summary of them here:
-
-A secondary goal to providing a specification document could be to also provide
-an executable specification via tools such as the [K
-framework](http://www.kframework.org/index.php/Main_Page).  Providing such an
-executable specification may become a primary goal if it can be achieved
-without too much effort and without hampering the achievement of other goals.
-
-Through the application of rigorous development practices and the availability
-of a complete language specification, we hope to apply formal verification
-methods to the Fe compiler and also to programs generated by it.
-
-## Progress and roadmap
-
-Development of Fe is currently in its early stages.  Here's a rough
-overview of recently completed work as well as some short and long-term goals.
-None of these lists should be considered exhaustive and should also not be
-considered to represent the exact prioritization of work items.
-
-### Recently achieved goals
-
-* [x] Completed port of [python's stdlib `tokenize`
-  module](https://github.com/python/cpython/blob/2a58b0636d1f620f8a85a2e4c030cc10551936a5/Lib/tokenize.py)
-  to Rust
-* [x] Settled on overall design of parser
-* [x] Completed a few basic parsers
-* [x] All code is now tested on both native and WASM platforms with WASM
-  testing via node.js
-* [x] Achieve ~100% code coverage by tests.  Coverage is currently maximized at
-  ~99% due to some code paths being unreachable and our coverage tool currently
-  having no support for ignoring specific lines of code.
-* [x] Identify an appropriate subset of the Python grammar and any grammatical
-  extensions needed to define a Fe EBNF grammar
-  - Eliminate certain Pythonic grammatical elements that don't apply to Fe
-    (such as `async` constructs, `lambda` expressions, argument packing syntax
-    like `*args` and `**kwargs`, etc.)
-  - Include grammatical elements that more directly capture Fe language
-    constructs (such as event, contract, or interface definitions, type
-    dimensions, etc.)
-* [x] Finish writing parsers for the Fe EBNF grammar
-* [x] Parse a basic greeter contract to AST
-* [x] Compile the AST of a basic guest book contract to YUL
-* [x] Generate YUL source code from YUL AST objects (via [yultsur](https://github.com/axic/yultsur/))
-* [x] Invoke the Solidity compiler with generated YUL source to produce a
-  compiled binary (via [solc-rust](https://github.com/axic/solc-rust))
-* [x] Implement a functional [ERC20 token contract](https://github.com/ethereum/fe/blob/00121f5ec06e46fcca261ad4d340f43e7183c2c7/compiler/tests/fixtures/erc20_token.fe) in Fe and compile it.
-
-### Short-term goals
-
-* [ ] Implement a functional Uniswap Demo in Fe
-
-### Long-term goals
-
-* Implement compilation routines for the full set of current Fe features
-* Implement a YUL compiler in Rust to eliminate the need to interact with the
-  Solidity compiler
-* Formally verify the Fe compiler using the language specification
```

Comment:
There's a lot of deleted stuff but I felt most is just clutter that will make it harder for people to get to know the language. If you think some of this should be preserved then maybe it could be moved to another document that we would link to.